include/stdexec/__detail/__basic_sender.hpp

```diff
@@ -444,10 +461,10 @@ namespace stdexec {
     static constexpr auto connect = __detail::__connect;
     static constexpr auto start = __detail::__start;
     static constexpr auto complete = __detail::__complete;
-    static constexpr auto get_completion_signagures = __detail::__get_completion_signagures;
+    static constexpr auto get_completion_signagures = __detail::__get_completion_signatures;
```

Comment:
Might want to fix this "signagures" to "signatures", too.

Reply:
```suggestion
    static constexpr auto get_completion_signatures = __detail::__get_completion_signatures;
```